package.json

```diff
@@ -368,11 +368,11 @@
                 },
                 {
                     "command": "appService.StartRemoteDebug",
-                    "when": "config.appService.enableRemoteDebugging == true"
+                    "when": "never"
```

Comment:
"never" means these won't show up in the command palette, but you already work with the command palette. Just remove these 8 lines

Reply:
Got it, yeah that makes sense.